node/nodeutil/node.go

```diff
@@ -0,0 +1,34 @@
+package nodeutil
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SetNodeReady sets the node ready condition to true
```

Comment:
Did we already have something else like this before? Where was it?

Reply:
We did not.

Reply:
So I added this so the test suite can determine if the node is ready vs waiting on pod being able to read from /stats/summary (which isn't technically accurate) and thought it would be a good utility. You can see it in use in the cmd code.